proof/invariant-abstract/RealTime_AI.thy

```diff
@@ -449,6 +449,10 @@ lemma valid_irq_states_release_queue_update[simp]:
   "valid_irq_states (release_queue_update f s) = valid_irq_states s"
   by (simp add: valid_irq_states_def)
 
+lemma valid_irq_states_reprogram_timer_update[simp]:
+  "valid_irq_states (reprogram_timer_update f s) = valid_irq_states s"
+  by (simp add: valid_irq_states_def)
+
 (* FIXME: move to Invariants_AI *)
```

Comment:
```suggestion
(* FIXME RT: move to Invariants_AI *)
```
(I know not your fault, but if we're here already we should fix the FIXME tags so we can find things later *)